examples/lib/stories/rendering/text_example.dart

```diff
@@ -101,3 +132,20 @@ class MyTextBox extends TextBoxComponent {
     super.render(canvas);
   }
 }
+
+class MyScrollTextBox extends ScrollTextBoxComponent {
+  MyScrollTextBox(
+    String text, {
+    required super.size,
+    super.boxConfig,
+    super.position,
+    super.anchor,
+  }) : super(text: text, textRenderer: _box);
+
+  @override
+  void render(Canvas canvas) {
+    final rect = Rect.fromLTWH(0, 0, width, height);
+    canvas.drawRect(rect, Paint()..color = Colors.white10);
```

Comment:
Create this paint in `onLoad` or in the constructor
(so that people don't copy paste this code and accidentally creates 60 new paint objects per second)